src/paho/mqtt/client.py

```diff
@@ -24,8 +24,14 @@
 HAVE_SSL = True
 try:
     import ssl
+    import sys
     cert_reqs = ssl.CERT_REQUIRED
-    tls_version = ssl.PROTOCOL_TLSv1
+
+    # Since 3.6 this constant selects highest protocol
+    if sys.hexversion > 0x03060000:
```

Comment:
This check is too specific since there's support in other Python version. Use `hasattr` instead.